department-of-veterans-affairs/vets-website
# 90448 consistency for each summary list item, better alignment with mocks

## Summary
- Fix bug with inconsistent markup and styling for list items on the TE summary pages. In addition to visual fix, this should be an improvement for screen readers too.
- Set color for subtitle to better align with mocks

team: @department-of-veterans-affairs/dbex-trex 
toggle cleanup: department-of-veterans-affairs/va.gov-team#65089
## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#90448

## Testing done
**Manual test setup**
1. Enable toggle `disability_526_toxic_exposure`
2. Start a new claim with new conditions
3. On the toxic-exposure/conditions page, select at least one new condition
4. Continue to herbicide or additional-exposures pages
5. Select one or more checkboxes and also enter text in the 'other' input box
6. Continue to the summary page for that TE question
7. Verify each list item has consistent markup and spacing

## Screenshots
<img width="1253" alt="Screenshot 2024-08-14 at 2 50 22 PM" src="https://github.com/user-attachments/assets/b055d4a2-cb92-4f4d-af46-a23ad0b32458">


## What areas of the site does it impact?
526ez

## Acceptance criteria

### Quality Assurance & Testing
No new tests added to validate markup, but all existing unit and e2e passed

- [n/a] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [n/a] Documentation has been updated ([link to documentation](#) \*if necessary)

### Error Handling

- [n/a] Events are being sent to the appropriate logging solution
- [n/a] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [n/a] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
